otwcode/otwarchive
# AO3-5761 Add Rack Attack for basic rate limiting

## Issue

https://otwarchive.atlassian.net/browse/AO3-5761

## Purpose

Adds Rack Attack middleware with basic configuration for rate limiting.

## Testing Instructions

Generate a huge number of requests and see what happens. Otherwise, it's a fairly isolated change.
